sdk/trace/src/testIncubating/java/io/opentelemetry/sdk/trace/TracerConfigTest.java

```diff
@@ -158,4 +161,67 @@ private static Stream<Arguments> tracerConfiguratorArgs() {
         Arguments.of(enableStartsWithD, scopeDog, enabled()),
         Arguments.of(enableStartsWithD, scopeDuck, enabled()));
   }
+
+  @Test
+  void setScopeConfigurator() {
```

Comment:
New test to ensure changes reach `ExtendedTracer#isEnabled()` and have the desired affect.